src/model/quotes.js

```diff
@@ -740,15 +744,15 @@ class QuotesModel {
 
       // make an error callback
       let opts = {
-        method: 'PUT',
+        method: Enum.Http.RestMethods.PUT,
         url: fullCallbackUrl,
         data: JSON.stringify(fspiopError.toApiErrorObject()),
         // use headers of the error object if they are there...
         // otherwise use sensible defaults
-        headers: this.generateRequestHeaders(headers || {
-          'fspiop-destination': fspiopSource,
-          'fspiop-source': 'switch',
-          'fspiop-http-method': 'PUT'
+        headers: (headers || {
+          'Enum.Http.Headers.FSPIOP.DESTINATION': fspiopSource,
+          'Enum.Http.Headers.FSPIOP.SOURCE': 'Enum.Http.Headers.FSPIOP.SWITCH.value',
+          'Enum.Http.Headers.FSPIOP.HTTP_METHOD': 'Enum.Http.RestMethods.PUT'
         }, true)
       }
       let res
```

Comment:
Please use the shared library sendRequest function else you have to keep the generateRequestHeaders function